docs/design/storage_replication.md

```diff
@@ -0,0 +1,39 @@
+# Objective 
+
+The purpose of this document is to gather ideas and function for the use of storage volumes with replication controllers.  Once the requirements are established a code design will be added.  
+
+The current design allows only one volume to map to a replicated pod. Its desirable when scaling an application that each replica receives its own unique volume.
+
+# Goals
+
+* Determine how a replication controller should map replicas to volumes
+* Determine how volumes are handled when an RC scales up or down.
+* Define the code design and markup changes required for Replication Controllers to assign unique volumes to each replica
+
+# Assigning Volumes to Replicas
+There are two potential volume type assignments for replication controllers- existing unclaimed volumes and non-existant dynamically provisioned volumes.
+
```

Comment:
Also shared storage.  The current implementation only supports shared due to the single claimName in the PodTemplate, which is good for shared.  We'd need to preserve this functionality while also allowing a distinct volume per replica.